test/local/remove_route_test.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var removeRoute = require('../../routes/remove_route')
+var test = require('../ptaptest')
+
+test(
+  'remove_route removes a route from a list of routes',
+  function (t) {
+    var routeList = [
+      { path: '/route_1' },
+      { path: '/route_2' },
+      { path: '/route_3' }
+    ]
+
+    removeRoute(routeList, '/unknown_route')
+    t.equal(routeList.length, 3)
+
+    removeRoute(routeList, '/route_3')
+    t.equal(routeList.length, 2)
```

Comment:
Add a check to ensure the correct route is removed, not just that the count is correct.